Source/WebCore/bridge/objc/objc_utility.mm

```diff
@@ -195,8 +196,8 @@ JSValue convertObjcValueToValue(JSGlobalObject* lexicalGlobalObject, void* buffe
     switch (type) {
         case ObjcObjectType: {
             id obj = *(const id*)buffer;
-            if ([obj isKindOfClass:[NSString class]])
-                return convertNSStringToString(lexicalGlobalObject, (NSString *)obj);
+            if (auto *str = dynamic_objc_cast<NSString>(obj))
+                return convertNSStringToString(lexicalGlobalObject, str);
             if ([obj isKindOfClass:webUndefinedClass()])
```

Comment:
I don't think it makes much sense here because we won't be using the result of `dynamic_objc_cast`. We could use `is_objc` but I don't think that'll really buy us anything.